cms/djangoapps/api/docs/decisions/001-course-apps-api.rst

```diff
@@ -0,0 +1,178 @@
+Course Apps API
+_______________
+
+Status
+======
+Proposal
+
+Context
+=======
+
+The new `Course Authoring MFE`_ includes a new UX called "Pages and Resources"
+for configuring course "apps" such as progress, wiki, teams, discussions,
+etc. all from one place. Currently, these apps are intermingled with the
+core platform and are inseparable from it, however, future apps could be
+implemented as plugins.
+
+This ADR proposes a way to configure these apps from the backend that MFEs
+can use and potentially enable/disable these apps using an API.
+
+.. _Course Authoring MFE: https://github.com/edx/frontend-app-course-authoring/
+
+
+Decision
+========
+
+We can introduce a new type of Open edX plugin, a course app. These are
+course-level apps that add new functionality to the course for students.
+Each course app has associated metadata and a standard API to determine if
+the app is available, enabled, etc.
+
+This metadata should be made available via a configuration class similar to
+that used for Django apps, or course tabs etc. We can develop a new course
```

Comment:
See here => `We can develop a new course apps API that can be driven by the metadata provided by these installed plugins.` 

We are using plugins and course apps interchangeably. Installed Plugins are actually Installed Apps. I think we'd need to set expectations here what is more preferable.

Reply:
The apps are a type of plugin, so they are plugins. So any installed app is also an installed plugin of type app.

Reply:
In any case I have rewritten most of the ADR to clarify things better.